components/match2/wikis/splitgate/match_summary.lua

```diff
@@ -0,0 +1,234 @@
+---
+-- @Liquipedia
+-- wiki=splitgate
+-- page=Module:MatchSummary
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local DisplayHelper = require('Module:MatchGroup/Display/Helper')
+local Logic = require('Module:Logic')
+local Lua = require('Module:Lua')
+local Table = require('Module:Table')
+local Template = require('Module:Template')
+
+local MatchGroupUtil = Lua.import('Module:MatchGroup/Util', {requireDevIfEnabled = true})
+local MatchSummary = Lua.import('Module:MatchSummary/Base', {requireDevIfEnabled = true})
+local OpponentDisplay = Lua.import('Module:OpponentDisplay', {requireDevIfEnabled = true})
+
+local _EPOCH_TIME = '1970-01-01 00:00:00'
+local _EPOCH_TIME_EXTENDED = '1970-01-01T00:00:00+00:00'
+
+local htmlCreate = mw.html.create
+
+local _GREEN_CHECK = '<i class="fa fa-check forest-green-text" style="width: 14px; text-align: center" ></i>'
+local _RED_CROSS = '<i class="fas fa-times cinnabar-text" style="width: 14px; text-align: center" ></i>'
+local _ICONS = {
+	check = _GREEN_CHECK,
+	cross = _RED_CROSS
+}
+local _NO_CHECK = '[[File:NoCheck.png|link=]]'
+local _LINK_DATA = {
+	vod = {icon = 'File:VOD Icon.png', text = 'Watch VOD'},
+	preview = {icon = 'File:Preview Icon.png', text = 'Preview'},
+	lrthread = {icon = 'File:LiveReport.png', text = 'LiveReport.png'},
+	siegegg = {icon = 'File:SiegeGG icon.png', text = 'SiegeGG Match Page'},
+}
+
+local CustomMatchSummary = {}
+
+function CustomMatchSummary.getByMatchId(args)
+	local match = MatchGroupUtil.fetchMatchForBracketDisplay(args.bracketId, args.matchId)
+
+	local frame = mw.getCurrentFrame()
+
+	local matchSummary = MatchSummary():init()
+	matchSummary.root:css('flex-wrap', 'unset') -- temporary workaround to fix height, taken from RL
+
+	matchSummary:header(CustomMatchSummary._createHeader(match))
+				:body(CustomMatchSummary._createBody(match))
+
+	-- comment
+	if match.comment then
+		local comment = MatchSummary.Comment():content(match.comment)
+		comment.root:css('display', 'block'):css('text-align', 'center')
+		matchSummary:comment(comment)
+	end
+
+	-- footer
+	local vods = {}
+	for index, game in ipairs(match.games) do
+		if game.vod then
+			vods[index] = game.vod
+		end
+	end
+
+	match.links.lrthread = match.lrthread
+	match.links.vod = match.vod
+	if not Table.isEmpty(vods) or not Table.isEmpty(match.links) then
+		local footer = MatchSummary.Footer()
+
+		-- Game Vods
+		for index, vod in pairs(vods) do
+			footer:addElement(Template.safeExpand(frame, 'vodlink', {
```

Comment:
not specific to this PR, but i think we should think about modulizing this template, we use it quite regularly now

Reply:
see #804